oio/cli/cluster/cluster.py

```diff
@@ -161,35 +172,32 @@ def get_parser(self, prog_name):
             metavar='<srv_type>',
             help='Service type.')
         parser.add_argument(
-            'srv_addr',
-            metavar='<srv_addr>',
-            help='Network address of the service.'
-        )
+            'srv_ids',
+            metavar='<srv_ids>',
+            nargs='+',
+            help='ID(s) of the services.')
         return parser
 
-    def _unlock_one(self, type_, addr):
-        service_info = {'type': type_, 'addr': addr}
-        try:
-            self.app.client_manager.cluster.unlock_score(service_info)
-            yield type_, addr, "unlocked"
-        except Exception as exc:
-            yield type_, addr, str(exc)
+    def _unlock_services(self, parsed_args):
+        srv_definitions = list()
+        for srv_id in parsed_args.srv_ids:
+            srv_definitions.append(
+                self.app.client_manager.cluster.get_service_definition(
+                    parsed_args.srv_type, srv_id))
+        for batch in _bounded_batches(srv_definitions, 4096):
```

Comment:
4096 is a bit unusual. Our other paginated requests usually build pages of 1000 elements. It may be a good idea to define that at the top of the file, or as a default value in `_bounded_batches`.